nordic-sdk/components/libraries/bootloader_dfu/bootloader_util_arm.c

```diff
@@ -88,6 +88,8 @@ static void bootloader_util_reset(uint32_t start_addr)
     asm(
         ".equ MASK_ONES,  0xFFFFFFFF\n\t" /* Ones, to be loaded into register as default value before reset.  */
         ".equ MASK_ZEROS, 0x00000000\n\t" /* Zeros, to be loaded into register as default value before reset. */
+        ".equ xPSR_RESET, 0x21000000\n\t" /* Default value of xPSR after System Reset. */
+        ".equ EXC_RETURN_CMD, 0xFFFFFFF9\n\t" /* EXC_RETURN for ARM Cortex. When loaded to PC the current interrupt service routine (handler mode) willl exit and the stack will be popped. Execution will continue in thread mode. */

```

Comment:
These were simply missing from the file, and were copied directly from the Keil ARMCC code in the same file.